README.md

```diff
@@ -8,10 +8,25 @@
 
 ## Introduction
 
-Supply Chain Goat follows the tradition of existing *Goat projects (e.g. OWASP Web Goat). It provides a training ground to practice implementing countermeasures specific to the software supply chain. StepSecurity defines a supply chain attack as an attack that tries to hijack software that you produce or consume.
+Supply Chain Goat follows the tradition of existing *Goat projects (e.g. OWASP Web Goat). It provides a training ground to practice implementing countermeasures specific to the software supply chain. 
+
+A software supply chain is anything that affects, modifies or goes into your code from development to production deployment through a CI/CD pipeline. A software supply chain includes: who wrote it, when it was contributed, how it’s been reviewed for security issues, known vulnerabilities, supported versions, license information, etc. A software supply chain also encompasses other parts of the stack beyond a single application, including the build and packaging scripts or the software running the infrastructure the application runs on. A software supply chain also includes any information you want to know about the software you’re running to help you determine any risks in running it.
+
+StepSecurity defines a supply chain attack as an attack that tries to hijack software that you produce or consume. [OSSF Scorecards](https://github.com/ossf/scorecard) recommends using the StepSecurity tool to secure workflows and fix GitHub token permissions.
```

Comment:
`recommends using the StepSecurity tool to remediate some of the checks`. Let us just leave it at that for now...